frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -126,31 +126,62 @@ function FileManagerMenu:setUpdateItemTable()
         checked_func = function() return self.ui.file_chooser.show_unsupported end,
         callback = function() self.ui:toggleUnsupportedFiles() end
     }
-    self.menu_items.items_per_page = {
-        text = _("Items per page"),
-        help_text = _([[This sets the number of items per page in:
+    self.menu_items.items = {
+        text = _("Items"),
```

Comment:
Maybe something like `View`? But that's just a synonym for `Display`, already used above when CoverBrowser is enabled. Without CoverBrowser you don't get access to settings like home abbreviation? It might be necessary to give that some attention, even if not necessarily as part of this PR. Also pinging @poire-z